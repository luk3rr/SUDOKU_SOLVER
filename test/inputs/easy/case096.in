000806009 090007308 007500000 800009051 000084006 070100000 400900800 000030000 705000200 
