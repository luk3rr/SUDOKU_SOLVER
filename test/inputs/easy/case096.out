Solving the following grid:
0 0 0 | 8 0 6 | 0 0 9 
0 9 0 | 0 0 7 | 3 0 8 
0 0 7 | 5 0 0 | 0 0 0 
------+-------+------
8 0 0 | 0 0 9 | 0 5 1 
0 0 0 | 0 8 4 | 0 0 6 
0 7 0 | 1 0 0 | 0 0 0 
------+-------+------
4 0 0 | 9 0 0 | 8 0 0 
0 0 0 | 0 3 0 | 0 0 0 
7 0 5 | 0 0 0 | 2 0 0 


Solution found :')

2 4 3 | 8 1 6 | 5 7 9 
5 9 1 | 4 2 7 | 3 6 8 
6 8 7 | 5 9 3 | 1 2 4 
------+-------+------
8 6 2 | 3 7 9 | 4 5 1 
1 5 9 | 2 8 4 | 7 3 6 
3 7 4 | 1 6 5 | 9 8 2 
------+-------+------
4 3 6 | 9 5 2 | 8 1 7 
9 2 8 | 7 3 1 | 6 4 5 
7 1 5 | 6 4 8 | 2 9 3 

Algorithm: BFS
Total time: 95 ms
Total expanded states: 17032
